src/joystream-lib/api.ts

```diff
@@ -1,11 +1,14 @@
+/* eslint-disable new-cap */
```

Comment:
I guess you could also remove this `eslint-disable` and do: 
```ts
import { Bytes, GenericEvent, Raw, Option, Vec, TypeRegistry, u64 as U64 } from '@polkadot/types'
```
but not sure what is better here. I personally feel uncomfortable if I see `ts-ignore` or `eslint-disable` :P

Reply:
Renamed :+1: